README.md

```diff
@@ -182,6 +178,14 @@ You can even override the name of the docker image with the `CHE_IMAGE_REPO` var
 bash -c "export CHE_IMAGE_REPO=rhche/che-server ; export CHE_IMAGE_TAG=nightly ; dev-scripts/minishift_deploy.sh"
 ```
 
+By default, the docker image of the deployed to Minishift with the Keycloak integration *disabled*.
```

Comment:
s/the docker image of the deployed to Minishfit with/Che is deployed on minishift with/